### ISSUE TITLE
Component Multi-Fixtures (Element type is invalid)

### ISSUE THREAD

**Issue by justinmacorin** [react-cosmos/react-cosmos#1640]
### What's wrong?

Basic component multi-fixtures failing. 

> Internal error: Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

### Steps to reproduce

1.  Create a new fixture.
```
javascript "use client";
import React from 'react';

export default {
  'Basic Alert': function MyFixture() {
    const [isOpen, setIsOpen] = React.useState(true);
    return <h1>Hello</h1>;
    }
};
```
4. Internal error: Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

### Screenshots

<img width="745" alt="image" src="https://github.com/react-cosmos/react-cosmos/assets/47508868/2c4f71ea-73d6-447f-b1fa-2293c2077650">
<img width="424" alt="image" src="https://github.com/react-cosmos/react-cosmos/assets/47508868/4ea16625-27ad-4304-8d1d-0c6c7397fc5a">
<img width="782" alt="image" src="https://github.com/react-cosmos/react-cosmos/assets/47508868/dbff4cf9-430b-47a7-9364-a0ca3ab89551">

### Useful info

    "next": "14.1.4",
    "npm": "^10.8.1",
    "react": "^18",
    "react-dom": "^18",
    "react-cosmos": "^6.1.1",
    "react-cosmos-next": "^6.1.1",

**Comment by ovidiuch** [comment]
Hi @justinmacorin,

Multi fixtures cannot be used in combination with Client components when using Next.js. This is a limitation mentioned here: https://reactcosmos.org/docs/getting-started/next#limitations

Next 14.2.x breaks Client fixtures completely in Cosmos. There is another issue opened about this that I've investigated but haven't found the time to create the fix yet. There is a chance that once fixed this limitation will go away but I can't say for sure until implemented.

**Comment by ovidiuch** [comment]
Fixed Next 14.2.x compatiblity in https://github.com/react-cosmos/react-cosmos/issues/1639#issuecomment-2212393036 but unfortunately it didn't change anything related to this issue. Namely, exporting multi fixture objects in Client fixtures is still unsupported as per the current docs. I'll close this but feel free to start a Discussion if you have any ideas how to overcome this. Thanks.
